service/controller/deleter.go

```diff
@@ -49,29 +48,8 @@ func NewDeleter(config DeleterConfig) (*Deleter, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.K8sClient must not be empty", config)
 	}
 
-	if config.ProjectName == "" {
-		return nil, microerror.Maskf(invalidConfigError, "%T.ProjectName must not be empty", config)
-	}
```

Comment:
Why is this removed? Checked in a parent package?

Reply:
I remove these everywhere I find it. This is configured to the operatorkit controller and this is checking if the name is given or not. No reason to do it all the way through all the layers.

Reply:
I remove these everywhere I find it. This is configured to the operatorkit controller and this is checking if the name is given or not. No reason to do it all the way through all the layers.

Reply:
This is still confusing to me.

Reply:
We will soon use the `pkg/project` approach anyway. Don't waste your energy on this.

Reply:
I already approved the PR. I'm talking about general approach.